dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -1,8 +1,11 @@
+import base64
 import itertools
 import json
 from collections import defaultdict
 from typing import Iterator, Optional
 
+from Crypto.Cipher import AES
+from Crypto.Protocol.KDF import PBKDF2
```

Comment:
Can you throw a try/except `ImportError`/`HAS_CRYPTO` around these and deal with it appropriately in the passwords export? I.e. logging or raising error. Same in the firefox plugin.